File: src/main/java/net/adventuregame/bloom/BrightFilterShader.java
package net.adventuregame.bloom;

import com.chidozie.core.shaders.ShaderProgram;

public class BrightFilterShader extends ShaderProgram {
	
	private static final String VERTEX_FILE = "bloom/simpleVertexShader.vert";
	private static final String FRAGMENT_FILE = "bloom/brightFilterFragmentShader.frag";
	
	public BrightFilterShader() {
		super(VERTEX_FILE, FRAGMENT_FILE);
	}

	@Override
	protected void getAllUniformLocations() {	
	}

	@Override
	protected void bindAttributes() {
		super.bindAttribute(0, "position");
	}

}

File: src/main/kotlin/net/adventuregame/bloom/BrightFilterShader.kt
package net.adventuregame.bloom

import com.adv.core.shaders.ShaderProgram

class BrightFilterShader : ShaderProgram(VERTEX_FILE, FRAGMENT_FILE) {
    override fun getAllUniformLocations() {
    }

    override fun bindAttributes() {
        super.bindAttribute(0, "position")
    }

    companion object {
        private const val VERTEX_FILE = "bloom/simpleVertexShader.vert"
        private const val FRAGMENT_FILE = "bloom/brightFilterFragmentShader.frag"
    }
}
